setup.py

```diff
@@ -20,6 +20,17 @@
 
 from setuptools import setup
 from setuptools.extension import Extension
+
+# install requirements before import
+from setuptools import dist
+SETUP_REQUIRES = [
+    "setuptools >= 39.0",
+    "wheel >= 0.26.2",
```

Comment:
I'm not sure why wheel should be a requirement for building the library. Is that only required if you want to create a wheel?

We have #41, but I believe that requires building a wheel for ipopt first.